### ISSUE TITLE
BTS and Old Aperture Doors Item can be exploited

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#2218]
![Image](https://user-images.githubusercontent.com/12244376/30512732-f71cc5fe-9acb-11e7-9ef2-c71d3df3cec5.jpg)

**Comment by TeamSpen210** [comment]
Yes, this is the portal bumping bug. There's not much you can do about it, so it isn't too much of a problem.

**Comment by LautaroL20** [comment]
func_noportal_volume
